app/views/devise/mailer/password_change.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Hello <%= @resource.email %>!</p>
```

Comment:
Because I see this says @resource.email instead of email. Just wondering how it works for my own understanding, otherwise this all looks good.

Reply:
It seems to be pulling this from the Devise gem. There is a little bit of magic here, Devise is creating @resource somewhere else.